packages/picasso/src/Notification/Notification.tsx

```diff
@@ -117,6 +133,7 @@ const useStyles = makeStyles<Theme>(styles, {
 export const Notification = forwardRef<HTMLElement, PrivateProps>(
   function Notification(props, ref) {
     const { className, variant, elevated, ...rest } = props
+    const dataTestId = props['data-testid'] || 'Notification'
```

Comment:
```suggestion
    const dataTestId = props['data-testid']
```

In Picasso we should not have any hardcoded test id. We even have ticket in progress, that removes all hardcoded testids

Reply:
in another case, how to use a data-testid for a close button?